src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/runtime.Windows_NT.Microsoft.NETCore.Runtime.CoreCLR.props

```diff
@@ -11,13 +11,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <NativeBinary Include="$(BinDir)clretwrc.dll" />
-    <NativeBinary Include="$(BinDir)coreclr.dll" />
+    <NativeBinary Include="$(BinDir)clretwrc.dll" LinkerUsage="diagnostic" />
+    <NativeBinary Include="$(BinDir)coreclr.dll" LinkerUsage="runtime" />
     <NativeBinary Include="$(BinDir)dbgshim.dll" />
-    <NativeBinary Include="$(BinDir)mscordaccore.dll" />
-    <NativeBinary Include="$(BinDir)mscordbi.dll" />
-    <NativeBinary Include="$(BinDir)mscorrc.debug.dll" />
-    <NativeBinary Include="$(BinDir)mscorrc.dll" />
+    <NativeBinary Include="$(BinDir)mscordaccore.dll" LinkerUsage="diagnostic" />
+    <NativeBinary Include="$(BinDir)mscordbi.dll" LinkerUsage="diagnostic" />
+    <NativeBinary Include="$(BinDir)mscorrc.debug.dll" LinkerUsage="diagnostic" />
+    <NativeBinary Include="$(BinDir)mscorrc.dll" LinkerUsage="diagnostic" />
     <NativeBinary Include="$(BinDir)sos.dll" />
     <NativeBinary Include="$(BinDir)Redist\ucrt\DLLs\$(BuildArch)\*.dll" Condition="'$(BuildType)'=='Release' AND '$(BuildArch)' != 'arm64'" />
```

Comment:
Do the CRT files need to be included as well?

Reply:
Good question - my understanding is they're only needed for win7 and below. If that's the case, I'd prefer to use another mechanism to include them on that platform specifically (or maybe just disable native asset trimming by default when publishing for win7 and below).

Reply:
> disable native asset trimming by default when publishing for win7 and below

Publishing for win7 and below is the default. We try to stay away from specifying OS versions for publishing. You publish for Windows (e.g. `-r win-x64`) and the output works on all Windows versions supported by given .NET Core version.

I agree that it would be a good idea to have an option to drop the UCRT files for linked apps, but it is not obvious to me what the best way to enable it is.